src/lib/plugin.js

```diff
@@ -536,7 +536,9 @@ class FiveBellsLedger extends EventEmitter2 {
             relatedResources.cancellation_condition_fulfillment)
         } else if (fiveBellsTransfer.state === 'rejected') {
           const rejectedCredit = find(fiveBellsTransfer.credits, 'rejected')
-          const rejectionMessage = rejectedCredit ? rejectedCredit.rejection_message : 'transfer timed out.'
+          const rejectionMessage = rejectedCredit
+            ? new Buffer(rejectedCredit.rejection_message, 'base64').toString()
```

Comment:
Hmm. I think not using it though would expose the DoS vulnerability that caused that switch in the first place. What if I set the `rejection_message` to `10000000000000`?